### PR TITLE
Update ReleaseNotes to include Input Recording Service improvements

### PR DESCRIPTION
**Pull request by MaxWang-MS**
## Overview
Add sections related to Input Recording Service to release notes.

### CODE REVIEW

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -35,13 +35,21 @@ Two new solvers are ready for use with mainline MRTK.
 
   ![Directional Indicator Solver](Images/Solver/DirectionalIndicatorExampleScene.gif)
 
+### Input Recording Service improvements
+
+`InputRecordingService` and `InputPlaybackService` can now record and plays back eye gaze input. Recording has been optimized to ensure a consistent framerate throughout the recording period while recording file size and save time are also reduced by about 50%. Saving and loading of recording files can now be performed asynchronously. Note the file format of the recording has changed in this MRTK version, please see [here](InputSimulation/InputAnimationFileFormat.md) for more information on the new version 1.1 specifications.
```
Comment (davidkline-ms) [inline Documentation/ReleaseNotes.md:40]:
```suggestion
`InputRecordingService` and `InputPlaybackService` can now record and play back eye gaze input. Recording has been optimized to ensure a consistent framerate throughout the recording period while recording file size and save time are also reduced by about 50%. Saving and loading of recording files can now be performed asynchronously. Note the file format of the recording has changed in this MRTK version, please see [here](InputSimulation/InputAnimationFileFormat.md) for more information on the new version 1.1 specifications.
```

**Review by davidkline-ms — COMMENTED** [review]
Minor fix. Then happy to approve.

**Review by keveleigh — APPROVED** [review]

**Pull request merged**